src/Features/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -46,6 +46,7 @@ internal abstract class AbstractPullDiagnosticHandler<TDiagnosticsParams, TRepor
         protected const int DocumentDiagnosticIdentifier = 2;
 
         private readonly EditAndContinueDiagnosticUpdateSource _editAndContinueDiagnosticUpdateSource;
+        private readonly IGlobalOptionService _globalOptions;
```

Comment:
Curious why this couldn't apply to the context anymore?

Reply:
It could - but the fewer ways of getting the same thing the better.  This is a mef service so we should be retrieving it via MEF.